Makefile

```diff
@@ -238,3 +238,15 @@ catalog-build: opm ## Build a catalog image.
 .PHONY: catalog-push
 catalog-push: ## Push a catalog image.
 	$(MAKE) docker-push IMG=$(CATALOG_IMG)
+
+.PHONY : build-deploy
+KUBECTL = $(shell which kubectl)
+build-deploy : docker-build docker-push clear-bundle bundle bundle-build bundle-push catalog-build catalog-push
+	$(shell cat deploy/catalog_source.yaml| sed 's,image: .*,image: $(CATALOG_IMG),' > deploy/catalog_source_temp.yaml)
+	$(KUBECTL) create -n olm -f deploy/catalog_source_temp.yaml
+	rm -rf deploy/catalog_source_temp.yaml
+	$(KUBECTL) create ns argocd
+	$(KUBECTL) create -n argocd -f deploy/operator_group.yaml
+	$(KUBECTL) create -n argocd -f deploy/subscription.yaml
+clear-bundle :
+	rm -fr bundle/
```

Comment:
I am not sure we need to add a new target to combine build and deploy.  Please do `make help` and see if there are existing targets that suit your need. You will notice that targets are organized by `Dev`, `Build`, and `Deploy`. We are trying to keep the Makefile clean.

Reply:
It think it would be helpful for testing operator locally , by building and deploying on local cluster.

Reply:
Doesn't `make bundle` generates `deploy/catalog_source.yaml` with image = `$CATALOG_IMG` already?

Reply:
It creates bundle and and CSV , whereas this is convenient command to invoke all commands required to build and test locally, but coming to your previous comment this doesn't keep makefile cleaner , where each command is concerned with one thing build or deploy , we can close this.